pkg/manager/backup/mysql/handler/grpc/handler.go

```diff
@@ -241,7 +241,6 @@ func (s *server) RemoveIPs(ctx context.Context, req *payload.Backup_IP_Remove_Re
 func toBackupMetaVector(meta *model.MetaVector) (res *payload.Backup_Compressed_MetaVector, err error) {
 	return &payload.Backup_Compressed_MetaVector{
 		Uuid:   meta.GetUUID(),
-		Meta:   meta.GetMeta(),
```

Comment:
Also, it is better to modify the SQLs in `internal/db/rdb/mysql` ([e.g.](https://github.com/vdaas/vald/blob/3b9325b8bc44942708c9e137f702cafd1fc594b2/internal/db/rdb/mysql/mysql.go#L259)) and [DB schema](https://github.com/vdaas/vald/blob/3b9325b8bc44942708c9e137f702cafd1fc594b2/k8s/jobs/db/initialize/mysql/configmap.yaml).

The names of the functions in `pkg/manager/backup/mysql/service` should also be updated. (but it's too heavy to modify all...)

Reply:
I'll work on it, maybe we have to change backup/{mysql | cassandra}

Reply:
I fixed thease problem at #794